### ISSUE TITLE
Hidden elements are removed from dom

### ISSUE THREAD

**Issue by BasMulders** [petecoop/laravel-inky#7]
Hi! Thanks for your work on this package. Works pretty good. One thing I've noticed is that Emogrifier removes elements that are hidden by `display: none;`. When a preheader needs to be included, it needs to be present in the html of the email - even though it is hidden via css.

I've found that adding `$emogrifier->disableInvisibleNodeRemoval();` right after the `new Emogrifier` instance is created fixes this issue. But, I'm unaware of any other effects this might cause. Perhaps this has to be an option? I'm thinking that, in line with Foundation for Emails, the default behavior should be that `$emogrifier->disableInvisibleNodeRemoval();` is present.

I'd like your opinion on this issue. Thanks!

**Comment by petecoop** [comment]
Hey, think I'm going to replace emogrifier - see #3 

Does `display: none;` work in email clients? Looks like gmail and outlook don't support it: https://www.campaignmonitor.com/css/

**Comment by BasMulders** [comment]
Sorry for my late reply. What Foundation does under the hood is apply some CSS, `display: none;` is one of the rules. The whole CSS that is applied to a .preheader looks as follows:

`
span.preheader {
  display:none !important;
  visibility:hidden;
  mso-hide:all !important;
  font-size:1px;
  color: $body-background; // needs to match background color
  line-height:1px;
  max-height:0px;
  max-width:0px;
  opacity:0;
  overflow:hidden;
}
`

When I try to send an email using the default Foundation Email template (https://github.com/zurb/foundation-emails-template), the preheader works as it should (in Apple Mail, iOS mail and gmail). Then, when I try to send it via your package, it does not display the preheader properly. I tested some things and found that Emogrifier removes the `display: none;` - so I figured that's what causes the problem. I'm unsure why campaign monitor says `display: none;` is not supported, as my short experience tells me that it is in fact supported (in this case).

I'll check and report back if the problem still persists with the replacement of emogrifier. Thanks!
